### ISSUE TITLE
Production details

### ISSUE THREAD

**Issue by lihaksa** [Bemeier/bmek#2]
Dear dev,

I tried sending the Gerber file to my local Pcb manufacturer but they asked for the following details (they said these were missing):

Pcb Material                                    
Pcb Layer Account
Pcb Board Thickness                  
Copper Thickness
Inner Copper Thickness
Soldermask Color
Silkscreen Color
Surface Finish                                        
Panelization                    
Technical Border (Strip)                       
“Multilayer: Stack-up, Impedance Control, Via Filled Type etc.”

So sorry for being such a noob. Any help would be greatly appreciated.

Thanks so much in advance!

**Comment by jankolkmeier** [comment]
I highly recommend you to look up some resources explaining what all of these manufacturer terms mean before just blindly ordering something from a manufacturer. The best way to learn is to put in a little effort yourself first. That will also help you deal with other issues you might face down the line!

That being said, It's a simple 2 layer board, 1.5mm thickness, FR4 material likemost keyboard pcbs. Rest is up to you.

**Comment by jankolkmeier** [comment]
Actually, let me reopen this b/c I think it's a fair point that I should add this info to the documentation (even if it would be obvious to most,I did notice there are a couple of people that picked this as their first project)

**Comment by lihaksa** [comment]
Thank you so much.

**Comment by Bobthebearfood** [comment]
May be a stupid question because this is my first real custom, but when I search HRO-TYPE-C-31-M-12 on google, the result is an smd port, but in the picture, the port looks through hole. Which is it?

**Comment by jankolkmeier** [comment]
You're right, the pictures on the main page are from the prototype PCB (v2) and are using Wuerth part-through hole, part smd usb receptacles. The latest versions (v2.1 and v3) use the smd-only connector of the type that you're referring to (HRO-TYPE-C-31-M-12 )
